docs/source/blog/arc-collaboration.md

```diff
@@ -0,0 +1,90 @@
+
+- `plot_centroid_trajectory`, which creates a plot of the trajectory of a given keypoint (or the centroid of a collection of keypoints) with a single line of code. Introduced in [#394](https://github.com/neuroinformatics-unit/movement/pull/394).
+- `plot_occupancy`, which creates an occupancy plot of an individual, given its position data. Collections of individuals are aggregated over, and if multiple keypoints are provided, the occupancy of the centroid is calculated. Introduced in [#403](https://github.com/neuroinformatics-unit/movement/pull/403).
+
+Examples to showcase the use of these plotting functions are currently [being produced](https://github.com/neuroinformatics-unit/movement/issues/415).
+[Our other examples](https://movement.neuroinformatics.dev/examples/index.html) have also been updated to use these functions where possible.
+
+### Broadcasting Methods
+
+Most of the time during analysis of experimental data, we want to repeat the same action multiple times across different coordinates, or individuals, or time-points.
+The data-structures that `movement` relies on - and the analysis methods it provides - go a long way to making this kind of analysis easy, thanks to array broadcasting.
+However when it comes to writing your own, custom analysis functions, most of the time it is much easier to write a function that takes in a single input, rather than writing an appropriate function that acts on an entire `DataArray`.
+But this approach typically means that you're then stuck writing a `for` loop over the values in your `DataArray`, and on top of that having to copy across the dimension labels or coordinates into the new array you're making as the `for` loop runs.
+
+With this in mind, we have decided to expose the {func}`movement.utils.broadcasting.make_broadcastable` decorator, which can turn functions that take in one-dimensional data (e.g., a single spatial coordinate, or a single time-point) into functions that act across `DataArray`s with appropriate dimensions.
+You can find a new {ref}`example <sphx_glr_examples_broadcasting_your_own_methods.py>` on how to use this decorator on your own functions.
+
+### Regions of Interest (RoIs)
+
+One of the biggest feature introductions to `movement` during this period is {mod}`support for RoIs <movement.roi>`.
+We now support one-dimensional regions of interest (segments, or piecewise-linear structures formed of multiple segments) as well as two-dimensional polygonal regions.
+You can create RoIs using the {class}`movement.roi.LineOfInterest` and {class}`movement.roi.PolygonOfInterest` classes respectively.
+There are more details about how the class is implemented on top of [`shapely`](https://shapely.readthedocs.io/en/stable/) in the {class}`docstring <movement.roi.base.BaseRegionOfInterest>`, and a note about how the interior of 2D shapes is handled.
+
+RoIs support the calculation of certain quantities from experimental data.
+Highlights include;
```

Comment:
Again, is the semicolon intentional or should it be a colon?

Reply:
I think it started out as a sentence with several clauses before I just bit the bullet and made it into a list (like many of the lists in this post!)